src/GitHub.Api/Authentication/LoginManager.cs

```diff
@@ -106,7 +109,7 @@ class LoginManager : ILoginManager
             var host = loginResultData.Host;
             var keychainAdapter = keychain.Connect(host);
             var username = keychainAdapter.Credential.Username;
-            var password = keychainAdapter.Credential.Token;
```

Comment:
Is this change needed? The username and password are stored in the keychain at the beginning of the process (in https://github.com/github-for-unity/Unity/pull/691/files#diff-6d6481eca7dfd1acee9426efc4be8854R75), and we always use them until we're finished with 2fa, right?

Reply:
I have to test a bit more to properly address this question, but this change is necessary as a result of the changes above.

Without the change above the user interface would think the user is authenticated at this point. In order to prevent that I intentionally did not save the token in the keychain. Which is why I need to pull the token from the previous login result.

Reply:
Okay yea, we just need to not save the keychain.